blaze-server/src/test/scala/org/http4s/server/blaze/BlazeServerSpec.scala

```diff
@@ -17,6 +21,20 @@ class BlazeServerSpec extends ServerAddressSpec {
 
       true must_== true
     }
+
+    "Startup with a custom ExecutionContext" in {
+      val s = BlazeBuilder
+        .bindAny()
+        .withExecutionContext(
+          ExecutionContext.
+            fromExecutorService(
+              Executors.newFixedThreadPool(5)))
```

Comment:
It would be a terribly verbose test for a parameter change, but to go red-to-green, an execution context that simply records that work was done, a request to the server, and then an assertion on that recording would do the trick, I think.